src/components/molecules/Wallet/Account.module.css

```diff
@@ -18,6 +16,7 @@
 .button,
 .address {
   color: var(--color-primary);
+  transition: 0.3s ease-in-out;
```

Comment:
thought(blocking): we should probably use a css variable for transition(-duration) - I remember seeing the same values in another file above.